build.gradle

```diff
@@ -106,7 +106,7 @@ configure(subprojects) { p ->
 
   repositories {
 	mavenLocal()
-	if (version.endsWith('BUILD-SNAPSHOT')) {
+	if (version.contains('.BUILD-SNAPSHOT') || version.contains('-SNAPSHOT')) {
```

Comment:
```suggestion
	if (version.endsWith('-SNAPSHOT')) {
```

Reply:
the feature that allows us to build snapshot jars from unmerged branches without polluting the main published snapshots now produces version in the format `MAJOR.MINOR.PATCH-SNAPSHOT-customQualifier`, hence the switch to `contains`... but at the same time I can fully drop support for `BUILD-SNAPSHOT` I guess

Reply:
do we still do branch pushes? I'd rather prefer Jitpack for it :)

Reply:
how about `if (version.endsWith('-SNAPSHOT') || version.contains('-SNAPSHOT-')) { //classic or customized snapshots` @bsideup ?

Reply:
Does not make me love Jitpack less but fine :D